src/lib/components/Badge.tsx

```diff
@@ -0,0 +1,14 @@
+import styled from 'lib/theme'
+
+import Row from './Row'
+
+export const Badge = styled(Row)<{ borderRadius?: number; padding?: string }>`
+  background-color: ${({ theme }) => theme.outline};
+  border-radius: ${({ borderRadius }) => `${borderRadius ?? 0.5}em`};
+  padding: ${({ padding }) => padding ?? '0.25em 0.375em'};
+  z-index: 2;
```

Comment:
This should probably use OVERLAY from lib/theme/layer.ts. It should not be defined inline, it should use a Layer constant.